mne_bids/mne_bids.py

```diff
@@ -225,6 +233,14 @@ def _participants_tsv(raw, subject_id, group, fname, verbose):
         Name of group participant belongs to.
     fname : str
         Filename to save the participants.tsv to.
+    write_mode : str, one of ('append', 'overwrite', 'error')
+        How the file should be handled if is exists already.
+        Defaults to 'append'.
+        If write_mode == 'append' the previously existing file will have the
```

Comment:
previously existing file can be just existing file. I feel that the duplicate entries will be overwritten should get a line return and be made a separate sentence for clarity and to highlight that action

Reply:
Something like?
```
        If write_mode == 'append' the existing file will have the
        new data added to it.
        If a duplicate entry is found it will be replaced with the new value
        provided.
```

Reply:
yeah, that's better